clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/lock/LockManagerImpl.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.clustered.client.internal.store.lock;
+
+import org.ehcache.clustered.client.internal.store.ClusterTierClientEntity;
+import org.ehcache.clustered.client.internal.store.ServerStoreProxyException;
+import org.ehcache.clustered.common.internal.messages.EhcacheEntityResponse;
+import org.ehcache.clustered.common.internal.messages.EhcacheEntityResponse.LockSuccess;
+import org.ehcache.clustered.common.internal.messages.ServerStoreOpMessage.LockMessage;
+import org.ehcache.clustered.common.internal.messages.ServerStoreOpMessage.UnlockMessage;
+import org.ehcache.clustered.common.internal.store.Chain;
+import org.ehcache.impl.internal.concurrent.ConcurrentHashMap;
+
+import java.util.Collections;
+import java.util.Set;
+import java.util.concurrent.TimeoutException;
+
+import static org.ehcache.clustered.common.internal.messages.EhcacheResponseType.LOCK_FAILURE;
+
+public class LockManagerImpl implements LockManager {
+
+  private final ClusterTierClientEntity clientEntity;
+  private final Set<Long> locksHeld = Collections.newSetFromMap(new ConcurrentHashMap<>());
```

Comment:
This set is maintained on all invocations but not used anywhere.

Reply:
done